common/src/main/java/com/wynntils/models/character/SkillPointModel.java

```diff
@@ -52,13 +57,20 @@ public class SkillPointModel extends Model {
     private static final String EMPTY_ACCESSORY_SLOT = "§7Accessory Slot";
     private static final int CHARACTER_INFO_SOUL_POINT_SLOT = 62;
     private static final int TOME_MENU_SOUL_POINT_SLOT = 89;
+    private static final Pattern SET_PATTERN = Pattern.compile("§a(.+) Set §7\\((\\d)/\\d\\)");
+    private static final String SET_BONUS_HEADER = "§aSet Bonus:";
+    private static final Pattern BONUS_SKILL_POINT_PATTERN =
+            Pattern.compile("§[ac]([+-]\\d+) §7(Strength|Dexterity|Intelligence|Defence|Agility)");
```

Comment:
Since we have the TestRegex setup now, should we be adding all future patterns to it?

Reply:
We don't necessarily test every regex, but yes, if we want tests, we do it there, not regexr.